From 4085caa42ae746c2f186e5e14d495bf33fd208b7 Mon Sep 17 00:00:00 2001
From: L4cache <42721712+L4cache@users.noreply.github.com>
Date: Wed, 11 Dec 2024 21:53:11 +0800
Subject: [PATCH] Edit CMakeLists

This patch updates VVdeC version and bypasses some problems during build process and not configurable.
---
 CMakeLists.txt        | 2 +-
 heifio/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bf0d8d06..00939d18 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -471,7 +471,7 @@ if (WITH_HEADER_COMPRESSION OR WITH_UNCOMPRESSED_CODEC)
         message("zlib not found")
     endif()
 
-    find_package(Brotli)
+    # find_package(Brotli)
     if (Brotli_FOUND)
         message("Brotli found")
         list(APPEND REQUIRES_PRIVATE "libbrotlidec")

diff --git a/heifio/CMakeLists.txt b/heifio/CMakeLists.txt
index 920eeca5..46010dc9 100644
--- a/heifio/CMakeLists.txt
+++ b/heifio/CMakeLists.txt
@@ -25,7 +25,7 @@ target_compile_definitions(heifio
         LIBHEIF_EXPORTS
         HAVE_VISIBILITY)
 
-find_package(TIFF)
+# find_package(TIFF)
 if (TIFF_FOUND)
     target_sources(heifio PRIVATE decoder_tiff.cc decoder_tiff.h encoder_tiff.h encoder_tiff.cc)
     target_link_libraries(heifio PRIVATE TIFF::TIFF)
-- 
2.50.1

